client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -12,28 +16,232 @@ import { connect } from 'react-redux';
-		siteId && fetch( siteId );
+		// Save settings request
+		if ( ! this.props.saveSettingsRequest && nextProps.saveSettingsRequest ) {
+			const settingsClean = omit( nextProps.settings, [
+				'save',
+				'isSaving',
+				'error',
+				'invalidValue',
+			] );
+			const areSettingsValid = validateSettings( settingsClean );
+			if ( nextProps.loaded ) {
+				if ( ! areSettingsValid ) {
+					nextProps.emailSettingsInvalidValue(
+						nextProps.siteId,
+						'Email Settings are invalid: please correct.'
+					);
+					nextProps.errorNotice( translate( 'Please correct your Email settings and try again.' ) );
+				} else {
+					nextProps.submit( nextProps.siteId, nextProps.settings );
+				}
+			} else {
+				nextProps.emailSettingsInvalidValue(
+					nextProps.siteId,
+					'Email Settings not loaded: please wait.'
+				);
+			}
+		}
+
+		// Settings save request finished.
+		if ( ! nextProps.isSaving && this.props.isSaving ) {
+			if ( nextProps.submitError ) {
+				nextProps.errorNotice(
+					translate( 'There was a problem saving the email settings. Please try again.' )
+				);
+			} else {
+				nextProps.successNotice( translate( 'Email settings saved.' ), { duration: 4000 } );
+			}
+		}
 	}
 
+	onChange = event => {
+		const { onChange, siteId } = this.props;
+		onChange( siteId, event );
+	};
+
+	renderOriginNotification = ( item, index ) => {
+		const { settings, loading } = this.props;
+		return (
+			<NotificationsOrigin
+				key={ index }
+				item={ item }
+				isPlaceholder={ loading }
+				recipient={ get( settings, [ item.field, item.option, 'value' ], '' ) }
+				placeholder={ get( settings, [ item.field, item.option, 'default' ], '' ) }
+				onChange={ this.onChange }
+				checkEmail={ item.checkEmail }
+			/>
+		);
+	};
+
+	renderInternalNotification = ( item, index ) => {
+		const { settings, loading } = this.props;
+		return (
+			<InternalNotification
+				key={ index }
+				item={ item }
+				checked={ 'yes' === get( settings, [ item.field, 'enabled', 'value' ], '' ) }
+				recipient={ get( settings, [ item.field, 'recipient', 'value' ], '' ) }
+				placeholder={ get( settings, [ item.field, 'recipient', 'default' ], '' ) }
+				isPlaceholder={ loading }
+				onChange={ this.onChange }
+			/>
+		);
+	};
+
+	renderCustomerNotification = ( item, index ) => {
+		const { settings, loading } = this.props;
+		return (
+			<CustomerNotification
+				key={ index }
+				item={ item }
+				isPlaceholder={ loading }
+				checked={ 'yes' === get( settings, [ item.field, 'enabled', 'value' ], '' ) }
+				onChange={ this.onChange }
+			/>
+		);
+	};
+
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
 		return (
 			<div className="email-settings__container">
+				<ExtendedHeader label={ translate( 'Origin' ) } />
+				<List>{ originNotifications.map( this.renderOriginNotification ) }</List>
+				<div>
+					<ExtendedHeader
+						label={ translate( 'Internal notifications' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **51** times:
`translate( 'notifications' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-settings-ui-save)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).